Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/QuantitySelector.kt

```diff
@@ -0,0 +1,77 @@
+package com.example.jetsnack.ui.components
+
+import androidx.compose.foundation.Text
+import androidx.compose.foundation.layout.ChainStyle
+import androidx.compose.foundation.layout.ConstraintLayout
+import androidx.compose.foundation.layout.preferredWidthIn
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.AddCircleOutline
+import androidx.compose.material.icons.outlined.RemoveCircleOutline
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.ui.tooling.preview.Preview
+import com.example.jetsnack.R
+import com.example.jetsnack.ui.theme.JetsnackTheme
+
+@Composable
+fun QuantitySelector(
+    count: Int,
+    updateCount: (Int) -> Unit,
```

Comment:
Personally, I'd expose two lambdas: `increaseItemCount` and `decreaseItemCount` and hoist the +/- 1 logic up